spinalcordtoolbox/scripts/sct_maths.py

```diff
@@ -426,9 +438,24 @@ def apply_array_operation(data, dim, arg_name, arg_value, parser):
     return data_out
 
 
+def contains(small, big):
+    """Check if the 'big' list contains the 'small' list, and return the start/end indexes if found."""
+    for i in range(len(big)-len(small)+1):
+        for j in range(len(small)):
+            if big[i+j] != small[j]:
+                break
+        else:
+            return i, i+len(small)
+    return False
+
+
 # MAIN
 # ==========================================================================================
 def main(argv: Sequence[str]):
+    # ensure '-volumewise 1' is parsed before any of the 'ParseDataOrScalarArgument' args
+    if idx := contains(['-volumewise', '1'], argv):
+        argv = ['-volumewise', '1'] + list(argv)[:idx[0]] + list(argv)[idx[1]:]
```

Comment:
...because `-volumewise` must be parsed before any of the array operation arguments.

Reply:
It looks like this constraint may be fixed by the proposal from @mguaypaq in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4485#issuecomment-2195052430, so my hacky fix here will no longer be needed. :tada:

Reply:
This is no longer an issue, since we no longer need to parse `-volumewise` first (thanks to `ActionTodo`).